sunspec/core/modbus/mbmap.py

```diff
@@ -22,12 +22,17 @@
 """
 
 import struct
+import sys
 
 try:
     import xml.etree.ElementTree as ET
 except:
     import elementtree.ElementTree as ET
 
+# Python 3 compatibility for long()
+if sys.version_info > (3,):
+    long = int
```

Comment:
Looks like it should work. I'll ask Bob why he used `long()` instead of `int()` when he wrote the code to see if there's something specific I'm missing. Otherwise it looks like switching to `int()` should be fine.